src/Microsoft.Diagnostics.Runtime/src/Common/ClrArray.cs

```diff
@@ -111,6 +111,22 @@ public ClrArray(ulong address, ClrType type)
         /// <returns>An <see cref="int"/> hash code for this instance.</returns>
         public override int GetHashCode() => Address.GetHashCode();
 
+        public int GetLowerBound(int dimension) => throw new NotImplementedException(nameof(this.GetLowerBound));
+
+        public int GetUpperBound(int dimension) => throw new NotImplementedException(nameof(this.GetUpperBound));
+
+        // -or- GetValue
+        public T ReadValue<T>(int index) where T : unmanaged => throw new NotImplementedException(nameof(this.ReadValue));
```

Comment:
I'm not sure if `Get` or `Read` is more clear.  For now I think using `GetValue` is right to be consistent with `ClrObject`, but if we decide to change it to Read later we should update `GetField` and similar as well.